site/diagrams/sequenceDiagrams/tenant_RemoveTenant_Success.txt

```diff
@@ -0,0 +1,9 @@
+title Remove Tenant (success)
+
+Client->+Tenant Service: transfer("tenant", "remove", tenant-id)
```

Comment:
...transfer("tenant", "remove", "${tenant_id}" )

is how we have it in the most sequence diagrams now. Can you please adopt the diagram?

Reply:
fixed.